syntax = "proto3";

message ArTemperatura {
    float temperatura = 1;
}

service Ar {
    rpc ligarAr(ArTemperatura) returns (ArTemperatura) {}
    rpc desligarAr(ArTemperatura) returns (ArTemperatura) {}
}